pyproject.toml

```diff
@@ -24,11 +24,13 @@ classifiers = [
 ]
 dependencies = [
     "dbt-adapters>=0.1.0a1,<2.0",
-    "psycopg2>=2.9,<3.0",
+    "psycopg2-binary>=2.9,<3.0",
     # installed via dbt-adapters but used directly
     "dbt-common>=0.1.0a1,<2.0",
     "agate>=1.0,<2.0",
 ]
+[project.optional-dependencies]
+no-binary = ["psycopg2>=2.9,<3.0"]
```

Comment:
I tried this locally and I wind up with both `psycopg2` and `psycopg2-binary` instead of just `psycopg2`.

Reply:
This now works.

```bash
pip install -e .
```
- Installs `psycopg2-binary` and does not install `psycopg2`
```bash
DBT_PSYCOPG2_NAME=psycopg2 pip install -e .
```
- Installs `psycopg2` and does not install `psycopg2-binary`